build.sbt

```diff
@@ -342,16 +346,33 @@ lazy val scioCassandra3: Project = Project(
   scioTest % "it"
 ).configs(IntegrationTest)
 
-lazy val scioElasticsearch: Project = Project(
-  "scio-elasticsearch",
-  file("scio-elasticsearch")
+lazy val scioElasticsearch2: Project = Project(
+  "scio-elasticsearch2",
+  file("scio-elasticsearch2")
+).settings(
+  commonSettings,
+  description := "Scio add-on for writing to Elasticsearch",
+  libraryDependencies ++= Seq(
+    "com.google.guava" % "guava" % guavaVersion,
+    "joda-time" % "joda-time" % jodaTimeVersion,
+    "org.elasticsearch" % "elasticsearch" % elasticsearch2Version
+  )
+).dependsOn(
+  scioCore,
+  scioTest % "test"
+)
+
+lazy val scioElasticsearch5: Project = Project(
+  "scio-elasticsearch5",
+  file("scio-elasticsearch5")
 ).settings(
   commonSettings,
   description := "Scio add-on for writing to Elasticsearch",
   libraryDependencies ++= Seq(
     "com.google.guava" % "guava" % guavaVersion,
     "joda-time" % "joda-time" % jodaTimeVersion,
-    "org.elasticsearch" % "elasticsearch" % elasticsearchVersion
+    "org.elasticsearch" % "elasticsearch" % elasticsearch5Version,
```

Comment:
@Alevy01 I think you can drop this one also. The only thing required is the Client.